app/src/picovr/cpp/DeviceDelegatePicoVR.cpp

```diff
@@ -227,6 +241,10 @@ struct DeviceDelegatePicoVR::State {
       }
     }
   }
+
+  int32_t GazeModeIndex() {
+    return type == kTypeNeo2 ? 2 : 1;
+  }
```

Comment:
Is this right? The G2 actually has two controllers, one for left and one for right and only one is ever visible. I would think the gaze index would be 2 for both G2 and Neo2.

Reply:
I wasn't aware of this, makes sense now seeing the controller settings where you can change to left/right mode. I've updated to not override the second controller.